.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": ["react", "es2015", "stage-0"],
```

Comment:
Not pretty sure for know.
And .. I can't understand 'I think it's better to name those and include those.'.
It's mean that I only get what I need?